adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -594,14 +567,14 @@ def ping(self) -> list[int]:
         self.logger.debug("Sending PINGREQ")
         self._sock.send(MQTT_PINGREQ)
         ping_timeout = self.keep_alive
-        stamp = self.get_monotonic_time()
+        stamp = ticks_ms()
         self._last_msg_sent_timestamp = stamp
         rc, rcs = None, []
         while rc != MQTT_PINGRESP:
             rc = self._wait_for_msg()
             if rc:
                 rcs.append(rc)
-            if self.get_monotonic_time() - stamp > ping_timeout:
+            if ticks_diff(ticks_ms(), stamp) > ping_timeout * 1000:
```

Comment:
make this consistent with the rest of the changes:
```suggestion
            if ticks_diff(ticks_ms(), stamp) / 1000 > ping_timeout:
```